packages/ui/src/types.ts

```diff
@@ -74,7 +75,22 @@ export interface PopupPositionElement {
     h: HorizontalAlignment
   }
 }
+
 export type PopupPosAlignment = 'right' | 'top' | 'float' | 'account' | 'full' | 'content' | 'middle'
+
+export function isPopupPosAlignment (x: any): x is PopupPosAlignment {
+  return (
```

Comment:
Minor: might be better?
```suggestion
  return ['right', 'top', 'float', 'account', 'full', 'content', 'middle'].includes(x)
```
or even
```suggestion
const POPUP_POS_ALIGMENTS = ['right', 'top', 'float', 'account', 'full', 'content', 'middle'] as const
export type PopupPosAlignment = typeof POPUP_POS_ALIGMENTS[number]
export function isPopupPosAlignment (x: any): x is PopupPosAlignment {
  return POPUP_POS_ALIGMENTS.includes(x)
}
```

Reply:
Let's leave as is.